code-challenges.test.js

```diff
@@ -67,8 +146,14 @@ var vowelTester2 = "academy"
 var vowelTester3 = "challenge"
 // Expected output: 2
 
+//tried to create a new variable as an array to see if it would work this way but still didnt work. 
+//let firstVIndex= ['learn', 'academy', 'challenge']
 
-
-
-
+// tried to use filter to make the strings strings into arrays but the test still 
 // b) Create the function that makes the test pass.
+
+const firstVIndex = (string) => {
+    let array = string.split("")
+    console.log(array)
+    return array.filter()
```

Comment:
good moves and good logic! Filter takes its own arguments and needs its own returns as well. best to set it to a variable and then "return" the variable